grasping_benchmarks_ros/graspnet_ros/graspnet_grasp_planner_service.py

```diff
@@ -156,15 +157,22 @@ def read_images(self, req : GraspPlannerRequest):
 
         return camera_data
 
+    def read_aruco_board(self, req : GraspPlannerRequest):
```

Comment:
Why is this function called at all if there is no need to filter the grasp candidates?